airbyte-integrations/connectors/source-shopify/source_shopify/schemas/orders.json

```diff
@@ -373,9 +373,9 @@
         "type": ["null", "string"]
       }
     },
-    "payment_terms": {
-      "type": ["null", "string"]
-    },
+"payment_terms": {
+  "type": ["null", "object"]
```

Comment:
Please refer to the official docs and declare the structure of the object:  https://shopify.dev/docs/api/admin-rest/2023-07/resources/order